core/src/main/java/org/mskcc/cbio/oncokb/util/AminoAcidConverterUtils.java

```diff
@@ -0,0 +1,32 @@
+package org.mskcc.cbio.oncokb.util;
+
+public class AminoAcidConverterUtils {
```

Comment:
Add  AminoAcidConverterUtilsTest under the test folder and test all methods under.